FieldworkAIDriver.lua

```diff
@@ -1385,6 +1385,11 @@ function FieldworkAIDriver:getAllFillLevels(object, fillLevelInfo)
 			if not fillLevelInfo[fillType] then fillLevelInfo[fillType] = {fillLevel=0, capacity=0} end
 			fillLevelInfo[fillType].fillLevel = fillLevelInfo[fillType].fillLevel + fillUnit.fillLevel
 			fillLevelInfo[fillType].capacity = fillLevelInfo[fillType].capacity + fillUnit.capacity
+			--used to check treePlanter fillLevel
```

Comment:
Don't change this now but I think we should come up with something for these special cases, a TreeplanterAIDriver may be an overkill, but that's what really is, a derived class with two overriden functions.